internal/otgutils/print.go

```diff
@@ -78,3 +78,71 @@ func LogPortMetrics(t testing.TB, otg *otg.OTG, c gosnappi.Config) {
 	out.WriteString("\n\n")
 	t.Log(out.String())
 }
+
+// LogLagMetrics is displaying otg lag stats.
+func LogLagMetrics(t testing.TB, otg *otg.OTG, c gosnappi.Config) {
+	t.Helper()
+	var out strings.Builder
+	out.WriteString("\nOTG LAG Metrics\n")
+	for i := 1; i <= 120; i++ {
+		out.WriteString("-")
+	}
+	out.WriteString("\n")
```

Comment:
```suggestion
	fmt.Fprintln(&out, strings.Repeat("-", 120))
```

Like what you did on line 55.

Reply:
Done